clusterloader2/pkg/util/podstore.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This file is copy of https://github.com/kubernetes/kubernetes/blob/master/test/utils/pod_store.go
+with slight changes regarding labelSelector and flagSelector applied.
```

Comment:
Is the only reason for copying over this file instead of vendoring is to make that change?

Reply:
Not a only reason. I also didn't want to vendor whole `https://github.com/kubernetes/kubernetes/blob/master/test/utils` due to having dependencies to kubernetes repository.

Reply:
Ok, that's reasonable for now.